io.catenax.id_based_comment/1.0.0/IdBasedComment.ttl

```diff
@@ -0,0 +1,184 @@
+#######################################################################
+# Copyright (c) 2023 BASF SE
+# Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Foerderung der angewandten Forschung e.V. (represented by Fraunhofer IML)
+# Copyright (c) 2023 Henkel AG & Co. KGaA
+# Copyright (c) 2023 Mercedes Benz AG
+# Copyright (c) 2023 SAP SE
+# Copyright (c) 2023 SupplyOn AG
+# Copyright (c) 2023 Volkswagen AG
+# Copyright (c) 2023 ZF Friedrichshafen AG
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.id_based_comment:1.0.0#> .
+
+:IdBasedComment a samm:Aspect ;
+   samm:preferredName "Id Based Comment"@en ;
+   samm:description "Aspect model for an exchange of comment belongig to a entity."@en ;
+   samm:properties ( :commentId :objectId [ samm:property :author; samm:optional true ] [ samm:property :postedAt; samm:optional true ] [ samm:property :changedAt; samm:optional true ] [ samm:property :commentText; samm:optional true ] [ samm:property :commentType; samm:optional true ] [ samm:property :requestDelete; samm:optional true ] [ samm:property :listOfReferenceDates; samm:optional true ] :objectType :customer :supplier ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:commentId a samm:Property ;
+   samm:preferredName "Comment ID"@en ;
+   samm:description "The Comment ID uniquely identifies the entity within the business relationship between a customer and its supplier."@en ;
+   samm:characteristic :UUIDv4IdTrait ;
+   samm:exampleValue "f5c151e4-30b5-4456-94fd-2a7b559b6121" .
+
+:objectId a samm:Property ;
+   samm:preferredName "Object ID"@en ;
+   samm:description "The ID of the object to which the comment belongs."@en ;
+   samm:characteristic :UUIDv4IdTrait ;
```

Comment:
you can make use of shared aspect model (io.catenax.shared.uuid:1.0.0) for UUID.
you can refer this https://github.com/eclipse-tractusx/sldt-semantic-models/blob/a07bdd5b1515767d4f8d07deb47cf2a4aa728bef/io.catenax.quality_message_content/1.0.0/QualityMessageContent.ttl#L35C9-L35C17